Source/DafnyTestGeneration/PathBasedModifier.cs

```diff
@@ -10,7 +10,7 @@ namespace DafnyTestGeneration {
   public class PathBasedModifier : ProgramModifier {
 
     // prefix given to variables indicating whether or not a block was visited
-    private const string blockVarNamePrefix = "$$visited$$_";
+    private const string BlockVarNamePrefix = "$$visited$$_";
```

Comment:
is this uppercase because it's a const?

Reply:
Yes, I think so. This originally came from [the other PR](https://github.com/atomb/dafny/pull/2), but that's what I expect was the motivation.